pkg/synchromanager/clustersynchro_manager.go

```diff
@@ -223,8 +233,8 @@ func (manager *Manager) reconcileCluster(cluster *clusterv1alpha2.PediaCluster)
 
 	config, err := buildClusterConfig(cluster)
 	if err != nil {
-		// TODO(iceber): update cluster status
 		klog.ErrorS(err, "Failed to build cluster config", "cluster", cluster.Name)
+		manager.UpdateClusterSynchroCondition(cluster.Name, clusterv1alpha2.InvalidConfigConditionReason, "invalid cluster config: "+err.Error(), metav1.ConditionFalse)
```

Comment:
If the cluster configuration is modified, but the configuration build fails, do we need to stop synchro？
I'm not sure

Reply:
When a user uses an invalid configuration, we assume that the user's changes have not been completed, so the invalid configuration should not affect the system.